index.js

```diff
@@ -22,7 +22,7 @@ function stubMethod(service, method, replacement) {
   if (!isStubbed(service)) stubService(service);
   if (!replacement) return sinon.stub(getService(service).prototype, method);
 
-  return sinon.stub(getService(service).prototype, method, function(params, callback) {
+  return sinon.stub(getService(service).prototype, method).callsFake(function(params, callback) {
```

Comment:
> since I've updated this module's usage of sinon stubs to use the .callsFake() method recommendation

Right, but downstream we're going to have tests that trip the warning via

```js
AWS.stub('S3', 'getObject', function() { ... });
```

There's also some convoluted `this` usage that some tests use inside that fake function that we should try to insure doesn't break. 

Don't sweat this if you have other things to move onto. Leave the PR sitting here as a note to future selves.